autotest/gdrivers/xmp.py

```diff
@@ -78,13 +78,13 @@ def test(self):
                 gdaltest.post_reason('opened with wrong driver')
                 print(ds.GetDriver().ShortName)
                 ret = 'failure'
-            elif self.expect_xmp and len(xmp_md) == 0:
+            elif self.expect_xmp and not xmp_md:
                 gdaltest.post_reason('did not find xml:XMP metadata')
                 ret = 'failure'
             elif self.expect_xmp and 'xml:XMP' not in ds.GetMetadataDomainList():
                 gdaltest.post_reason('did not find xml:XMP metadata domain')
                 ret = 'failure'
-            elif (not self.expect_xmp) and xmp_md is not None and len(xmp_md) != 0:
+            elif (not self.expect_xmp) and xmp_md is not None and xmp_md:
```

Comment:
I guess this instance of 'foo is not None and foo' is equivalent to 'foo' ? If so there are numerous such instances in this diff.

Reply:
Yeah, I will take care of these.